frontend/src/components/auth/Signup.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useContext, useState } from "react";
+import { Redirect } from "react-router-dom";
+// graphql {
+import { gql, useMutation } from "@apollo/client";
+// } graphql
+
+import authAPIClient from "../../APIClients/AuthAPIClient";
+import AuthContext from "../../contexts/AuthContext";
+import { AuthenticatedUser } from "../../types/AuthTypes";
+
+// graphql {
+const REGISTER = gql`
+  mutation Register(
```

Comment:
Learned from Alex that it's good practice to include the component name in the GraphQL operation name for ease of debugging (https://github.com/uwblueprint/starter-code-v2/pull/35#discussion_r624730384)
```suggestion
  mutation Signup_Register(
```